dags/prio_processor.py

```diff
@@ -67,6 +75,21 @@
     BUCKET_PRIVATE_B,
     "{{ var.value.prio_public_key_hex_internal }}",
     "{{ var.value.prio_public_key_hex_external }}",
+    env_vars={
+        # required for minio in the pod_mutation_hook
+        "PROJECT_ID": PROJECT_ADMIN,
+        "MINIO_ROOT_USER": username,
+        "MINIO_ROOT_PASSWORD": password,
```

Comment:
if you do go dedicated gke cluster route, a lot of these can be injected via secret to the containers